week1/woojin/index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style>
+        table, th, td {
+            border : 1px solid black;
+            border-collapse : collapse;
+        }
+        td {
+            width: 100px;
+            text-align: center;
+        }
+    </style>
+    <title>Document</title>
+</head>
+<body>
+    <h1>안녕하세요 :) </h1>
```

Comment:
```suggestion
    <h1>안녕하세요 :)</h1>
```
닫는 태그 앞의 공백은 제거해주는 것이 좋습니다!